mypy/main.py

```diff
@@ -188,6 +188,8 @@ def process_options(args: List[str],
                         help="dump type inference stats")
     parser.add_argument('--custom-typing', metavar='MODULE', dest='custom_typing_module',
                         help="use a custom typing module")
+    parser.add_argument('--custom-typeshed-dir', metavar='DIR',
+                        help="use the custom typeshed in DIR")
```

Comment:
I think Jukka had suggested to change this to "custom typeshed *package* in DIR" ?